bindings/python/src/session.cpp

```diff
@@ -748,6 +748,7 @@ void bind_session()
         )
 #endif
         .def("post_torrent_updates", allow_threads(&lt::session::post_torrent_updates))
+        .def("post_session_stats", allow_threads(&lt::session::post_session_stats))
```

Comment:
I don't think any example should use deprecated functions (and if it turns out to be really inconvenient, perhaps some functions shouldn't be deprecated, or their replacements should be fixed)